games-action/openjazz/openjazz-2017.12.02.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="A free, open-source version of classic Jazz Jackrabbit games"
+DESCRIPTION="A free, open-source version of classic Jazz Jackrabbit games.\
+Requirest the original game data to work. When launching OpenJazz, pass the location of the data, eg:\
+OpenJazz ~/jazz1
+"
+HOMEPAGE="https://github.com/AlisterT/openjazz"
+COPYRIGHT="Alister Thomson (alister_j_t at yahoo dot com)"
+LICENSE="GNU GPL v2"
+REVISION="1"
+COMMIT="0f59db3a6b0e2c076b39917dc65cfb3562611ee6"
+SOURCE_URI="https://github.com/AlisterT/openjazz/archive/$COMMIT.tar.gz"
+CHECKSUM_SHA256="55c82328ef3f0185951bd339be55411410d1fa786f391091e15b7a093a08fafd"
+SOURCE_DIR="openjazz-$COMMIT"
+PATCHES="disable_network.patch"
+
+ARCHITECTURES="?x86 ?x86_gcc2 x86_64 ?ppc ?arm"
+
+PROVIDES="
+	cmd:openjazz = $portVersion
+	"
+REQUIRES="
+        haiku
+		libsdl
+		libmodplug
+		zlib
+        "
+
+BUILD_REQUIRES="
+        haiku_devel
+		libsdl_devel
+		libmodplug_devel
+		zlib_devel
+		pkgconfig
+        "
+BUILD_PREREQUIRES="
+		cmd:gcc
+		cmd:make
+		"
+
+BUILD()
+{
```

Comment:
Try to add:
export LDFLAGS="-lnetwork"

If still not ok, try to add this TOO:
export CFLAGS="-D_BSD_SOURCE"